### PR TITLE
update to tag 3.0.5

### PR DESCRIPTION
**Pull request by katSchmid**

### CODE REVIEW

**Comment by chris-cooper** [comment]
It looks like there's some conflicts 
image

**Comment by katSchmid** [comment]
Are we good with node 16 or should i downgrade?

**Comment by chris-cooper** [comment]
> Are we good with node 16 or should i downgrade?

The `package.json` still says `"node": ">=6.0.0"` which is fine.  Are you seeing node 16 anywhere apart from the `.travis.yml` (which we won't be using)?

**Comment by katSchmid** [comment]
ah great, no more conflicts

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@ test
 .nyc_output
 doc/*
 !doc/gltf.png
+.github/*
```
Comment (chris-cooper) [inline .gitignore]: Just wondering why you're deleting/ignoring these as it looks like they're generated by `propeller-system` https://github.com/PropellerAero/gltf-pipeline/blob/98a7fc65f1852e91141ae155338fbca98149aec3/.github/dependabot.yml

Comment (katSchmid) [reply .gitignore]: oh, thats odd

Comment (chris-cooper) [reply .gitignore]: I think you missed the associated file deletes:
image

**Comment by katSchmid** [comment]
oh fun

**Pull request merged**